Darshanmakwana/ramweb
# Add a courses feature

This pull request introduces a new Courses feature to the website. The new feature allows users to view, filter, and select the courses. The following changes have been made:

- Our Courses: A new page has been added to display a list of available courses.
- Course Details: Each course includes details such as title, description.

Changes Made:
- Added new components for displaying the courses list and course details.
- Added routes for navigating between the courses list and course details pages.

### How to Test:
1. Navigate to the Courses section from the homepage or menu.
2. Ensure that the courses list loads correctly and is responsive.
3. Click on any course to view the details and ensure that the course information is correct.
